fido_app/models.py

```diff
@@ -24,19 +25,25 @@ class User(db.Model, UserMixin):
 
     # User info
     email = db.Column(db.String(80), unique=True, nullable=False)
-    display_name = db.Column(db.String(160), unique=False, nullable=False)
-    icon_url = db.Column(db.String(2083))
```

Comment:
`routes.py` still uses the `icon_url` field to create new Users on line 100, causing registration to fail when using passwords. Deleting/commenting out that line in `routes.py` allows everything to work fine, though.

Reply:
Good catch! Just fixed that in 7aecc16, so passwords should be working again. Feel free to merge this in if everything else looks good 👍🏻